src/primevue/tooltip/tooltip.ts

```diff
@@ -0,0 +1,31 @@
+import { TooltipDirectivePassThroughOptions } from "primevue/tooltip";
+import { tw } from "@/lib/tags.ts";
+
+const tooltip: TooltipDirectivePassThroughOptions = {
+  arrow: ({ context }) => {
+    const base = tw`absolute h-0 w-0 border-[7px] border-solid`;
+    const top = tw`-mr-[7px] border-b-0 border-t-gray-900 border-r-transparent border-l-transparent`;
```

Comment:
For what do we need the border, sometimes transparent sometimes gray-900? I guess the margin should be enough/ that we can get rid of this 7px border completely?

Reply:
I suppose I could add a comment here